### ISSUE TITLE
Research on how should data pipeline work

### ISSUE THREAD

**Issue by dancingsushii** [amosproj/amos2024ss04-building-information-enhancer#40]
## User story
1. As a Backend Developer I want to decide with other developers without feeling someone unsatisfied with the solution
2. I want / need {}
3. So that {}

## Acceptance criteria
* Some notes on the data pipeline specifications

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
